src/App.js

```diff
@@ -9,9 +14,13 @@ class App extends React.Component {
 
   render() {
     return (
-      <div className="App">
-        <Calculator />
-      </div>
+      <Layout>
+        <Routes>
+          <Route path="/home" element={<Home />} />
+          <Route path="/calculator" element={<CalculatorPage />} />
+          <Route path="/quote" element={<QuotePage />} />
+        </Routes>
+      </Layout>
```

Comment:
- [ ] Great Job setting up the routes, everything works perfectly, there's one small issue which is the root route '/' is empty and not assigned to any component, so it will cause an empty page when users visit your app, there's a lot of approaches to fix this what i suggest is changing
```JS
<Route path="/home" element={<Home />} />
```
to 

```JS
<Route path="/" element={<Home />} />
```

and update your home button (in the navbar) from 
```JS
 <NavLink className={classes.nav_link} to="/home">
            Home
          </NavLink>
```

To 

```JS
 <NavLink className={classes.nav_link} to="/">
            Home
          </NavLink>
```

Reply:
@ichala thanks a lot!